addon/components/bs-modal.js

```diff
@@ -407,6 +426,20 @@ export default Ember.Component.extend({
   _observeOpen: Ember.observer('open', observeOpen),
 
   /**
+   * Give the modal (or its autofocus element) focus
+   *
+   * @method takeFocus
+   * @private
+   */
+  takeFocus() {
+    let focusElement = this.$('[autofocus]').first();
```

Comment:
@bendemboski Did the autofocus feature actually work for you? The tests were fine, and were all passing using PhantomJS. However they failed in Chrome! And indeed the field was never focused in Chrome. 

Which seems right because the above jQuery selector does not select anything, as the autofocus field is not a child of the modal component with the DOM, because the modal-dialog is whormholed to another part of the DOM. This line should be like this:

        let focusElement = this.get('modalElement').find('[autofocus]');

I have a fix ready, however I was just wondering a) did it work for you?, an b) why tests were passing in PhantomJS but failing in Chrome... strange...